openedx/core/djangoapps/waffle_utils/testutils.py

```diff
@@ -0,0 +1,60 @@
+"""
+Test utilities for waffle utilities.
+"""
+
+from functools import wraps
+
+from waffle.testutils import override_flag
+
+
+def override_waffle_flag(flag, active):
```

Comment:
@andy-armstrong: I don't yet have a test for this decorator, but all other tests were added.